### PR TITLE
sep news + exports aticle update

### PR DESCRIPTION
**Pull request by anaisconce**

### CODE REVIEW

File: help/exports.md [hunk 1]
```diff
@@ -3,31 +3,31 @@
...
-Your browser will then download a CSV file containing a header row
-naming your columns, excluding any hidden columns, followed by all the
-rows of data in the table in text format.
+<center>![Export Table](../images/exports/export-table.png)</center>
 
-When on a page with multiple page widgets, "Export CSV" will export only the data in the
-currently-selected widget, to export all your data use "Export XLSX" option.
+Your browser will then download a file containing a header row
+naming your columns, excluding any hidden columns or filtered-out rows, followed by all the
+rows of data visible in the table. To export [filtered data](search-sort-filter.md#filtering), make sure to save your filter.
```
Comment (dsagal) [inline help/exports.md:14]: The last sentence is wrong I believe. One doesn't need to save filters to export filtered data. It looks like exporting linked views may fail to filter(?) but that won't be helped by saving a filter.

File: help/newsletters.md [hunk 2]
```diff
@@ -31,3 +31,4 @@ mill" means "useful experience, material, or knowledge", perfectly reflecting it
 - [June 2022 Newsletter](newsletters/2022-06.md): Filtering by range, new `PEEK()` function, and more!
 - [July 2022 Newsletter](newsletters/2022-07.md): Formula cheat sheet and summary tables in raw data!
 - [August 2022 Newsletter](newsletters/2022-08.md): Free team sites, conditional row styles, better formula help, and more!
+- [September 2022 Newsletter](newsletters/2022-09.md): Dark Mode. 🕶 + improved user management
```
Comment (dsagal) [inline help/newsletters.md:34]: Is that a stray period between "Dark Mode" and the sunglasses?

File: help/newsletters/2022-09.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+## Open Source Contributions 🙏
+
+Grist is [open source](https://github.com/gristlabs/grist-core){:target="\_blank"}. We’re grateful to the open source community for their contributions. Here are two recent highlights from a team that's not only using Grist, but is making it better for everyone. Thank you, @LouisDelbosc!
+
+### Improved User Management with Autocomplete
+
+When adding users to a document, the list will now autocomplete with emails of team members. This is currently available on grist-core and will land in hosted Grist by Monday, October 3rd. 
+
+![User Management with Autocomplete](../images/newsletters/2022-09/team-members-autocomplete.png)
```
Comment (dsagal) [inline help/newsletters/2022-09.md:119]: Should this be decorated with "screenshot-half"? It looks larger than normal.

Comment (anaisconce) [reply help/newsletters/2022-09.md:119]: Same issue here as usual. Screenshot half makes it too compressed and blurry. :( I can try retaking the screenshot at a different resolution.

**Review by dsagal — APPROVED** [review]
Thanks! The new screenshot is better.

For ideal screenshots in the future, if you can take it at exactly double resolution, then screenshot-half would make it perfect (and sharp on retina displays). Natalie has a way to do that on Windows. (On Macs with a retina screen, that happens by default.)

**Pull request merged**